src/amber/router/session.cr

```diff
@@ -1,6 +1,6 @@
 module Amber::Router::Session
   class Store
-    getter session_config : Hash(Symbol, String) = Amber::Server.settings.session
+    getter session_config : Hash(String, Int32 | String) = Amber::Server.settings.session
```

Comment:
I prefer symbols for keys.  Can we change this back to a symbol?

Reply:
I prefer symbols as well, I would use whatever is consistent across our api.